### PR TITLE
[RFC][UN-12633] uni-tooltip improvements

### PR DESCRIPTION
**Pull request by carantunes**
Related to: https://uniplaces.atlassian.net/browse/UN-12633

### CODE REVIEW

File: addon/components/uni-tooltip.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ export default Component.extend(ClickOutsideMixin, {
   isAlternative: false,
   yieldContent: false,
 
+  wrapperSelector: window,
```
Comment (AndreJoaquim) [inline addon/components/uni-tooltip.js]: This should placed with the above content; new line above

File: addon/components/uni-tooltip.js [hunk 2]
```diff
@@ -31,7 +32,9 @@ export default Component.extend(ClickOutsideMixin, {
     this._hideTooltip();
   },
 
-  click() {
+  click(ev) {
+    ev.stopPropagation();
```
Comment (AndreJoaquim) [inline addon/components/uni-tooltip.js:35]: Are we sure we want this all the time?

File: addon/styles/partials/_uni-tooltip.scss [hunk 3]
```diff
@@ -6,7 +6,7 @@
   }
 
   &:not(.uni-tooltip--alternative) {
-    @media (max-width: 767px) {
+    @include breakpoint("x-medium") {
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-tooltip.scss:9]: Is this the same? @mmelo @asantos00

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

File: addon/components/uni-tooltip.js (hunk 2 above)
Comment (carantunes) [reply addon/components/uni-tooltip.js:35]: I think it makes sense, if a tooltip is placed above another clickable component, say an offer card, only the tooltip should trigger an event. What the rest of the guys think?

File: addon/styles/partials/_uni-tooltip.scss (hunk 3 above)
Comment (mmelo) [reply addon/styles/partials/_uni-tooltip.scss:9]: `
$breakpoints: (
 "x-small" : 400px,
 "small"  : 600px,
 "medium" : 900px,
 'x-medium': 1100px,
...
`

Comment (carantunes) [reply addon/styles/partials/_uni-tooltip.scss:9]: 1100 is a our limit to mobile. anyone as a problem with this?

Comment (mmelo) [reply addon/styles/partials/_uni-tooltip.scss:9]: none, just stating that 767px of max-with is not the same as x-medium, but if it produces the expected results, I'm ok with it :)

**Review by AndreJoaquim — APPROVED** [review]

**Pull request merged**